intro-HPC/ch_troubleshooting.tex

```diff
@@ -29,6 +29,86 @@ \section{Out of quota issues}
 \fi
 % FIXME: Add how to request quota section
 
+\section{Issues connecting to login node}
+\label{sec:connecting-issues}
+
+If you're confused about the SSH public/private key pair concept, maybe the
+key/lock analogy in \autoref{subsec:how-do-ssh-keys-work} can help.
+
+If you have errors that look like
+
+\begin{prompt}
+%\userid{}%@%\loginnode%: Permission denied
+\end{prompt}
+
+
+
+If you are experiencing problems with connecting, here is a list of things to do
+that should help:
+
+\begin{enumerate}
+    \item It can take up to an hour for you VSC account to become active after
```

Comment:
`Keep in mind that it can take ... approved; until then, logging it to your VSC account will not work.`